### PR TITLE
Make sure traefik runs in an openshift workspace

### PR DESCRIPTION
**Pull request by ghost**
### What does this PR do?

Sets CHE_API for Traefik according to infra, adds sufficient permissions for tomcat and traefik

### CODE REVIEW

File: dockerfiles/dev/Dockerfile [hunk 1]
```diff
@@ -60,12 +62,17 @@ RUN mkdir /home/user/traefik ;\
     wget -O /home/user/traefik/traefik "https://github.com/containous/traefik/releases/download/v1.4.3/traefik_linux-amd64"; \
     chmod +x /home/user/traefik/traefik
 COPY traefik.toml /home/user/traefik/
+RUN    sudo chgrp -R 0 ~/traefik;\
+	sudo chmod -R g+rwX ~/traefik
```
Comment (benoitf) [inline dockerfiles/dev/Dockerfile]: can we group RUN instructions ?

File: dockerfiles/dev/Dockerfile [hunk 2]
```diff
@@ -60,12 +62,17 @@ RUN mkdir /home/user/traefik ;\
...
 
 
 RUN sudo mkdir /var/run/sshd
 RUN sudo  ssh-keygen -t rsa -f /etc/ssh/ssh_host_rsa_key -N '' && \
     sudo  ssh-keygen -t rsa -f /etc/ssh/ssh_host_ecdsa_key -N '' && \
     sudo  ssh-keygen -t rsa -f /etc/ssh/ssh_host_ed25519_key -N '' && \
     npm install -g typescript@2.5.3 typescript-language-server@0.1.4
-
+ADD entrypoint.sh /home/user/entrypoint.sh
+RUN sudo chmod +x /home/user/entrypoint.sh
```
Comment (benoitf) [inline dockerfiles/dev/Dockerfile]: do we need to chmod the file if file is committed with execute permissions ?

File: dockerfiles/dev/Dockerfile [hunk 3]
```diff
@@ -60,12 +62,17 @@ RUN mkdir /home/user/traefik ;\
...
 WORKDIR /projects
+ENTRYPOINT ["/home/user/entrypoint.sh"]
+CMD tail -f /dev/null
```
Comment (benoitf) [inline dockerfiles/dev/Dockerfile]: there is a missing new line

File: dockerfiles/dev/entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (benoitf) [inline dockerfiles/dev/entrypoint.sh:1]: copyright header is missing

**Review by benoitf — APPROVED** [review]

**Review by skabashnyuk — APPROVED** [review]

**Review by riuvshin — APPROVED** [review]

**Pull request merged**